wallet/src/repository/wallet/mod.rs

```diff
@@ -302,8 +304,12 @@ where
         let account = state.account;
 
         // Total amount of state and db transactions
-        let total = state.transaction_next_id;
-        let mut transactions = Vec::with_capacity(total as usize);
+        let total = state.transaction_next_id + u32::try_from(state.local_movements.len()).unwrap();
+        let mut transactions: Vec<BalanceMovement> = Vec::with_capacity(total as usize);
```

Comment:
should be easier cast transaction_next_id to usize instead of local_movements to u32 and later to usize?